designs/ProjectBoards.md

```diff
@@ -0,0 +1,32 @@
+# Using GitHub Project Boards for Customer Visibility and Tracking
+
+Our project boards will need to be created and accessed at the organizational level, not in any one repo (i.e. not in Home), because this is a restriction on project boards--if they're in a repo, they will be locked only to issues from that repo. Here's where ours will live: [https://github.com/orgs/NuGet/projects](https://github.com/orgs/NuGet/projects)
+
+We can use a simple or extended kanban column sets with automations (these are simply a template choice when a board is created), as we can see used in projects in other organizations:
+- https://github.com/dotnet/msbuild/projects
+- https://github.com/dotnet/roslyn/projects
+
+...and there's nothing stopping us from a complete rework of columns for any given project if that's helpful. Let's keep in mind simplicity where we can, and kanban columns are a pretty well-known approach. Here's a very simple example with automations, and additional status columns could be added if useful:
+
+![](../resources/ProjectBoards/kanban.png)
+
+Automations can extend to reviewing columns as well, e.g.:
+
+![](../resources/ProjectBoards/fullkanban.png)
+
+## A project per feature area
+Because issues can belong to multiple projects, and because automation in project boards will cause issues to move through various kanban columns, the overhead is low in tracking them separate locations.
```

Comment:
I would say this can be more simplified too.

Vulnerabilities Audit (PVA) is a single e2e feature that has components in VS, CLI, and NuGet.org. We can track that in a kanban board and have a single project?

Reply:
We sure could. Splitting them up was discussed in the meeting which is why I did it. I can recombine - all tasks would be cobbled together.